src/scss/components/_previous-link.scss

```diff
@@ -0,0 +1,17 @@
+@use 'microscope-sass/lib/bem';
+
+.openforms-previous-link {
+  @include bem.modifier('start') {
+    margin-block-end: 20px;
+    display: var(--of-previous-link-display-start, none);
+  }
+
+  @include bem.modifier('end') {
+    display: var(--of-previous-link-display-end, inline);
+  }
+
+  @include bem.element('icon') {
+    display: var(--of-previous-link-icon-display, none);
+    margin-inline-end: 10px;
```

Comment:
preferably apply a `display: flex; flex-direction: row; gap: 8px;` for horizontal spacing instead of relying on margins, it's more consistent with the rest of the code.

For (default) spacings, we also use multiples of `4px` (historical reasons)

Reply:
I've changed it 👍